torch/csrc/lazy/core/lazy_graph_executor.h

```diff
@@ -23,8 +23,10 @@ class TORCH_API LazyGraphExecutor {
 
   static LazyGraphExecutor* Get();
 
-  void RegisterTensor(std::shared_ptr<LazyTensor::Data> data);
-  void UnregisterTensor(LazyTensor::Data* data);
+  virtual ~LazyGraphExecutor();
+
+  virtual void RegisterTensor(std::shared_ptr<LazyTensor::Data> data);
```

Comment:
Since these become APIs now, maybe provide a few words explaining why people want to override these methods.